docs/_reference/auth.md

```diff
@@ -98,6 +98,53 @@ methods:
       }
       ```
 
+  - signature: auth.linkWithPopup(authProvider)
+    id: linkWithPopup
+    generator: true
+    description: Links the authenticated provider to the user account using a pop-up based OAuth flow.
+    arguments:
+      - name: authProvider
+      required: true
+      type: A [firebase.auth.AuthProvider](https://firebase.google.com/docs/reference/js/firebase.auth.AuthProvider) object.
+      description: The authentication provider to use for the request.
+    output: A [firebase.auth.UserCredential](https://firebase.google.com/docs/reference/js/firebase.auth#.UserCredential) instance.
+    example: |
+      ```javascript
+      const authProvider = new firebase.auth.GoogleAuthProvider();
+
+      function* linkSaga() {
+        try {
+          const data = yield(rsf.auth.linkWithPopup, authProvider);
+          yield put(linkSuccess(data));
+        } catch(error) {
+          yield put(loginFailure(error));
+        }
+      }
+      ```
+
+  - signature: auth.linkWithRedirect(authProvider)
+    id: linkWithRedirect
+    generator: true
+    description: Links the authenticated provider to the user account using a full-page redirect flow.
+    arguments:
+      - name: authProvider
+      required: true
+      type: A [firebase.auth.AuthProvider](https://firebase.google.com/docs/reference/js/firebase.auth.AuthProvider) object.
+      description: The authentication provider to use for the request.
+    output: A [firebase.Promise](https://firebase.google.com/docs/reference/js/firebase.Promise) containing void.
```

Comment:
The promise will be resolved by redux-saga so the saga will directly return `null`/`undefined`. You can remove this line. 😉